examples/typing/mg.jl

```diff
@@ -0,0 +1,60 @@
+using SemanticModels.Dubstep
+using Cassette
+using LightGraphs
+using MetaGraphs
+
+function escapehtml(i::AbstractString)
+    # Refer to http://stackoverflow.com/a/7382028/3822752 for spec. links
+    o = replace(i, "&" =>"&amp;")
+    o = replace(o, "\""=>"&quot;")
+    o = replace(o, "'" =>"&#39;")
+    o = replace(o, "<" =>"&lt;")
+    o = replace(o, ">" =>"&gt;")
+    return o
+end
+
+function savedot(io::IOStream, g::MetaDiGraph)
```

Comment:
Does this match the version at https://github.com/JuliaGraphs/MetaGraphs.jl/pull/71/files? Can you update that PR?